src/RadioButton/RadioButtonGroup.js

```diff
@@ -85,9 +85,8 @@ const RadioButtonGroup = React.createClass({
 
     if (nextProps.hasOwnProperty('valueSelected')) {
       newState.selected = nextProps.valueSelected;
+      this.setState(newState);
```

Comment:
Sorry, nitpick: Could we change this (and other similar scenarios previously fixed) to:

```js
if (nextProps.hasOwnProperty('valueSelected')) {
  this.setState({
    selected: nextProps.valueSelected
  });
}
```
That way the object allocation only occurs in the event the condition passes. :smile: